package.json

```diff
@@ -23,7 +23,7 @@
     "@jetbrains/babel-preset-jetbrains": "1.1.0",
     "@jetbrains/icons": "1.16.2",
     "@jetbrains/logos": "1.0.0",
-    "@jetbrains/ring-ui": "^1.0.0-beta.132",
+    "@jetbrains/ring-ui": "1.0.0-beta.132",
```

Comment:
For now is not working with the newest ring-ui due to https://youtrack.jetbrains.com/issue/RG-1846#focus=streamItem-27-3027937-0-0